core/sup_tree_core/system_metrics_reporter.ex

```diff
@@ -91,19 +91,28 @@ defmodule AntikytheraCore.SystemMetricsReporter do
   end
 
   defp log_too_many_messages_processes() do
-    ps = :recon.proc_count(:message_queue_len, 5)
-    [{_pid, top_len, _info} | _] = ps
+    infos = get_recon_infos_for_too_many_messages(@log_message_queue_length)
```

Comment:
https://dictionary.cambridge.org/ja/grammar/british-grammar/information
> Information is an uncountable noun

Reply:
Thank you.
On second thought, info/recon_info provides little extra context compared to the implementation itself.
So I changed the function and variable names.
https://github.com/access-company/antikythera/pull/267/commits/596aa5b59e5c8ba74f1f25d58b60096f059e08c1